imports/plugins/core/orders/client/containers/invoiceContainer.js

```diff
@@ -223,9 +223,9 @@ class InvoiceContainer extends Component {
     const paymentMethodId = orderBillingInfo.paymentMethod && orderBillingInfo.paymentMethod.paymentPackageId;
     const paymentMethodName = orderBillingInfo.paymentMethod && orderBillingInfo.paymentMethod.paymentSettingsKey;
     const paymentMethod = Packages.findOne({ _id: paymentMethodId });
+    const refund = _.get(paymentMethod.settings[paymentMethodName].support, "refund");
```

Comment:
Because you risk causing an error that would kill checkout I would rather we be extra cautious when testing for deeply nested properties like this. (although if `supports` doesn't exist you probably have other issues, but a runtime error is not the way to handle it.)